resources/CollectiblesRepentance.xml

```diff
@@ -1,441 +1,2967 @@
-    <entity ID="5" Image="Entities/Items/5.100.639 - Yuck Heart.png" Name="Yuck Heart" Subtype="639" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.640 - Urn of Souls.png" Name="Urn of Souls" Subtype="640" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.642 - Magic Skin.png" Name="Magic Skin" Subtype="642" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.650 - Plum Flute.png" Name="Plum Flute" Subtype="650" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.653 - Vade Retro.png" Name="Vade Retro" Subtype="653" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.655 - Spin to Win.png" Name="Spin to Win" Subtype="655" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.685 - Jar of Wisps.png" Name="Jar of Wisps" Subtype="685" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.687 - Friend Finder.png" Name="Friend Finder" Subtype="687" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.703 - Esau Jr.png" Name="Esau Jr" Subtype="703" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.704 - Berserk.png" Name="Berserk!" Subtype="704" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.705 - Dark Arts.png" Name="Dark Arts" Subtype="705" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.706 - Abyss.png" Name="Abyss" Subtype="706" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.709 - Suplex.png" Name="Suplex!" Subtype="709" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.710 - Bag of Crafting.png" Name="Bag of Crafting" Subtype="710" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.711 - Flip.png" Name="Flip" Subtype="711" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.712 - Lemegeton.png" Name="Lemegeton" Subtype="712" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.713 - Sumptorium.png" Name="Sumptorium" Subtype="713" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.714 - Recall.png" Name="Recall" Subtype="714" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.715 - Hold.png" Name="Hold" Subtype="715" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.719 - Keepers Box.png" Name="Keeper's Box" Subtype="719" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.720 - Everything Jar.png" Name="Everything Jar" Subtype="720" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.722 - Anima Sola.png" Name="Anima Sola" Subtype="722" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.723 - Spindown Dice.png" Name="Spindown Dice" Subtype="723" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.728 - Gello.png" Name="Gello" Subtype="728" Variant="100" />
-    <entity ID="5" Image="Entities/Items/5.100.729 - Decap Attack.png" Name="Decap Attack" Subtype="729" Variant="100" />
+    <entity
+      ID="5"
```

Comment:
can this be configured to only update leading/trailing whitespace? I don't really like how much vertical space this format takes up